public/app/plugins/panel/logs/LogsPanel.tsx

```diff
@@ -151,6 +213,36 @@ export const LogsPanel = ({
     [data]
   );
 
+  /**
+   * Scrolls the given row into view.
+   * Returns false if the reference to the logsContainerRef is not set.
+   */
+  const scrollIntoView = useCallback((row: HTMLElement): boolean => {
+    if (logsContainerRef.current && logsContainerRef.current.parentElement) {
+      const rowRect = row.getBoundingClientRect();
+      const containerRect = logsContainerRef.current.getBoundingClientRect();
+      const parentRect = logsContainerRef.current.parentElement.getBoundingClientRect();
+
+      console.log('row', rowRect, row);
+      console.log('container', containerRect, logsContainerRef.current);
+      console.log('parent', parentRect, logsContainerRef.current.parentElement);
+      console.log('--- --- ---');
+
+      // Both of these have the same bug, rows that are pinned near the end of the list do not scroll far enough
+      const scrollingTo = rowRect.top - containerRect.top;
+      // const scrollingTo = logsContainerRef.current.parentElement.scrollTop + rowRect.top - window.innerHeight / 2;
+      console.log('scrollingTo', scrollingTo);
+
+      logsContainerRef.current.parentElement.scroll({
```

Comment:
Was able to reproduce this bug only with smooth scrolling, and was able to reproduce it via scroll, and el.scrollIntoView.
Going to suggest we drop smooth scrolling here and in explore.